readthedocs/rtd_tests/tests/test_doc_builder.py

```diff
@@ -295,40 +271,11 @@ def test_append_conf_create_yaml(self, checkout_path, run):
             build_env=self.build_env,
             python_env=python_env,
         )
-        self.searchbuilder.append_conf()
-
-        run.assert_called_with('cat', 'mkdocs.yml', cwd=mock.ANY)
-
-        # There is a mkdocs.yml file created
-        generated_yaml = os.path.join(tmpdir, 'mkdocs.yml')
-        self.assertTrue(os.path.exists(generated_yaml))
-        config = yaml.safe_load(open(generated_yaml))
-        self.assertEqual(
-            config['docs_dir'],
-            os.path.join(tmpdir, 'docs'),
-        )
-        self.assertEqual(
-            config['extra_css'],
-            [
-                'http://readthedocs.org/static/css/badge_only.css',
-                'http://readthedocs.org/static/css/readthedocs-doc-embed.css',
-            ],
-        )
-        self.assertEqual(
-            config['extra_javascript'],
-            [
-                'readthedocs-data.js',
-                'http://readthedocs.org/static/core/js/readthedocs-doc-embed.js',
-                'http://readthedocs.org/static/javascript/readthedocs-analytics.js',
-            ],
-        )
-        self.assertIsNone(
-            config['google_analytics'],
-        )
-        self.assertEqual(
-            config['site_name'],
-            'mkdocs',
-        )
+        with pytest.raises(
+                MkDocsYAMLParseError,
+                match=MkDocsYAMLParseError.NOT_FOUND
```

Comment:
Nice, didn't know about this option in pytest!

Reply:
I learned it recently. :)